lib/ansible/modules/network/ios/ios_ping.py

```diff
@@ -143,7 +143,7 @@ def main():
     ping_results = run_commands(module, commands=results["commands"])
     ping_results_list = ping_results[0].split("\n")
 
-    success, rx, tx, rtt = parse_ping(ping_results_list[3])
+    success, rx, tx, rtt = parse_ping(ping_results_list[len(ping_results_list) - 1])
```

Comment:
Hi jmcgill298, for some reason my way and the simpler -1 method you suggest work fine but won't pass the unit tests.  My latest comment changes the way it checks determines which line it will parse to get the ping stats removing the need for a static list index.